src/mococrw/openssl_wrap.h

```diff
@@ -128,13 +129,22 @@ class OpenSSLException final : public std::exception
      * as message.
      *
      */
-    OpenSSLException() : _message{generateOpenSSLErrorString()} {}
+    OpenSSLException();
 
     const char *what() const noexcept override { return _message.c_str(); }
 
+    /**
+     * The following functions enable fine-tuned error handling via sub-library and reason
+     * information
+     */
+    const std::string &getLib() const { return _library; };
```

Comment:
Normally, we just return strings and not references. Returning references is potentially dangerous (even const ones). The overhead usually doesn't justify the memory management problems.

Reply:
Done.